dyndns53.go

```diff
@@ -156,11 +186,13 @@ func (rs *recordSet) upsert() (*route53.ChangeResourceRecordSetsOutput, error) {
 }
 
 func (rs *recordSet) validate() error {
-	if rs.name == "" {
-		return fmt.Errorf("missing record set name")
-	}
-	if !strings.HasSuffix(rs.name, ".") {
-		return fmt.Errorf(`record set name must end with a "."`)
+	for _, v := range rs.names {
+		if v == "" {
+			return fmt.Errorf("missing record set name")
```

Comment:
It's better to provide an index here since now there are many names. Something like:
```suggestion
			return fmt.Errorf("missing record set name at index %d", i)
```
Don't forget to change `_` to `i`.

Reply:
Done